src/client/components/PageLayout/Toolbar/Status/useRecipients.ts

```diff
@@ -0,0 +1,15 @@
+import { User } from '@meta/user'
+import { UserRoles } from '@meta/user/userRoles'
+
+import { useUsers } from '@client/store/ui/userManagement'
+import useGetUsers from '@client/pages/AssessmentHome/hooks/useGetUsers'
+
+import { StatusTransition } from './types'
+
+export const useRecipients = (props: { status: StatusTransition }): Array<User> => {
+  useGetUsers()
```

Comment:
maybe we should just invoke this hook only once in Country page? (and not in home + here) what do you think?